rest/model/conversions.go

```diff
@@ -46,6 +47,7 @@ func conversionFn(opts convertFnOpts, generatedConversions map[typeInfo]string)
 		OutType:  opts.outType,
 		OutIsPtr: opts.outIsPtr,
 	}
+	typeName := opts.nameOverride
```

Comment:
This is obviously just a style preference but in terms of readability, I think using opts.nameOverride everywhere instead of typeName is more clear.

Reply:
This was somewhat intentional because the function will change the variable. It's not an issue because opts isn't passed by reference, but I think this is clearer